server/lib/git.coffee

```diff
@@ -66,11 +66,13 @@ module.exports.init = (app, callback) ->
                     # available.
                     if not gitVersion? or \
                        compareVersions("1.7.10", gitVersion[1]) is 1
-                        commands.push "git clone #{url} #{app.name} && " + \
-                                      "cd #{app.dir} && " + \
-                                      "git branch #{branch} origin/#{branch} && " + \
-                                      "git checkout #{branch} && " + \
-                                      "git submodule update --init --recursive"
+                        command = "git clone #{url} #{app.name} && " + \
+                                  "cd #{app.dir} && "
+                        if branch isnt 'master'
```

Comment:
Not sure about that one. If the current branch is development we should be able to come back on the master branch.